Tuesday Tasks fixes (#20838)
DataDog/documentation: content/en/tracing/service_catalog/guides/streamlining-development-lifecycle-with-ci-visibility.md

To add or remove a pipeline associated to your service, click `Edit Metadata` in your service page, and go to Software Delivery. Please note that this is only available for [Service Catalog schema v2.2][7].

> {{< img src="tracing/service_catalog/service_catalog_delivery_lens.png" alt="Edit or add a pipeline in the UI" >}}

Reviewer: ```suggestion
{{< img src="tracing/service_catalog/service_catalog_delivery_lens.png" alt="The delivery view in the service catalog" >}}
```
